gobblin-utility/src/main/java/org/apache/gobblin/util/JobLauncherUtils.java

```diff
@@ -197,6 +197,14 @@ public static void cleanTaskStagingData(State state, Logger logger) throws IOExc
           throw new IOException("Clean up output directory " + outputPath.toUri().getPath() + " failed");
         }
       }
+
+      Path errPath = new Path(state.getProp(ConfigurationKeys.ROW_LEVEL_ERR_FILE));
```

Comment:
Is this cleanup related to sharding, or it is some separate improvement?

Also, is this error file not part of staging/output folder that are cleaned up earlier in this function?

Reply:
The error file is specified as a separate path from the writer output/staging dirs, so I considered it as another temporary job-level directory that has to be modified to work on the dataset-level. Hence why it has to be added in cleanup here.